zulipterminal/cli/run.py

```diff
@@ -252,18 +267,15 @@ def main(options: Optional[List[str]]=None) -> None:
         print('Zulip Terminal ' + ZT_VERSION)
         sys.exit(0)
 
-    if args.list_themes:
-        available_themes = all_themes()
-        print('Themes available:')
-        for theme in available_themes:
-            print('    {}'.format(theme))
-        sys.exit(0)
-
     if args.config_file:
         zuliprc_path = args.config_file
     else:
         zuliprc_path = '~/zuliprc'
 
+    if args.list_themes and not path.exists(path.expanduser(zuliprc_path)):
+        list_themes(('No theme', 'with no config'), False)
+        sys.exit(0)
```

Comment:
This code is moved around over the two commits, it should be fine where it was :)

The conditional can remain the same as before - I think we had confusion over the 'invalid' config file, I meant that `--list-themes` should be independent of whether we have a config file or not. For now it's fine to keep all the logic using args.list_themes in this top area.

Reply:
Yes, I think we can leave the user-preference part for now, for the reasons I gave in the review summary.

Reply:
I will update the function and squash the change with the previous commit then.

Reply:
That sounds good :+1: